google/ggrc-core
# GGRC-6634 Disable proposal creation

# Issue description

Prevent proposal and review creation for external models.

# Steps to test the changes

1. Login as "external_app@example.com"
2. Create Control
3. Login as "user@example.com"
4. Open created Control
5. Create proposal for Control
6. Should return error "Trying to create proposal for external model."
7. Request review for Control
8. Should return error "Trying to create review for external model."

# Solution description

Prevent proposal creation:
- Added validation to "instance_type" attribute, which checks if instance type is subclass of synchronizable.Synchronizable mixin.

Prevent review creation:
- Added validation to "reviewable_type" attribute, which checks if reviewable type is subclass of synchronizable.Synchronizable mixin.

# Sanity checklist




# PR Review checklist
